site/app/templates/grading/electronic/ta_status/StatusBase.twig

```diff
@@ -262,7 +262,7 @@
             {% if core.getUser().getGradingRegistrationSections()|length != 0 %}
                 {%  if team_assignment %}
                     <details class="btn btn-primary" id="details_button">
-                        <summary float="left">
```

Comment:
Did you mean to include these changes in this PR?

Reply:
@eddiekrystowski made those changes [here](https://github.com/Submitty/Submitty/pull/6747/commits/3831daaed1d44155873b356a08d62ea8b5b75445)  The changes were made to make it pass the tests I believe.

Reply:
Yes, it was failing an accessibility test since the float attribute is not allowed on the summary tag. Not sure why it was failing in this PR and not others.

Reply:
We were originally only checking accessibility on a  TA status page where there was no submissions is my guess.